tests/acceptance/conftest.py

```diff
@@ -13,202 +13,16 @@
-    )
-    parser.addoption(
-        "--test-variables",
-        action="store",
-        default="default",
-        help="configuration file holding settings for dedicated platform",
-    )
-    parser.addoption(
-        "--mender-image",
-        action="store",
-        default="default",
-        help="Mender compliant raw disk image",
-    )
-    parser.addoption(
-        "--commercial-tests",
-        action="store_true",
-        help="Enable tests of commercial features",
-    )
-
-
-def pytest_configure(config):
-    # ugly hack to access cli parameters inside common.py functions
-    global configuration
-
-    if config.getoption("--test-conversion"):
-        configuration["conversion"] = True
-    if config.getoption("--test-variables"):
-        configuration["test_variables"] = config.getoption("--test-variables")
-
-
-@pytest.fixture(scope="session")
-def host(request):
-    return request.config.getoption("--host")
-
-
-@pytest.fixture(scope="session")
-def user(request):
-    return request.config.getoption("--user")
-
-
-@pytest.fixture(scope="session")
-def ssh_priv_key(request):
-    return request.config.getoption("--ssh-priv-key")
-
-
-@pytest.fixture(scope="session")
-def http_server(request):
-    return request.config.getoption("--http-server")
-
-
-@pytest.fixture(scope="session")
-def board_type(request):
-    return request.config.getoption("--board-type")
-
-
-@pytest.fixture(scope="session")
-def sdimg_location(request):
-    return request.config.getoption("--sdimg-location")
-
-
-@pytest.fixture(scope="session")
-def qemu_wrapper(request):
-    return request.config.getoption("--qemu-wrapper")
-
-
-@pytest.fixture(scope="session")
-def mender_image(request):
-    return request.config.getoption("--mender-image")
-
-
-@pytest.fixture(scope="session")
-def bitbake_image(request):
-    return request.config.getoption("--bitbake-image")
-
-
-@pytest.fixture(scope="session")
-def conversion(request):
-    return request.config.getoption("--test-conversion")
-
-
-@pytest.fixture(scope="session")
-def use_s3(request):
-    return request.config.getoption("--use-s3")
-
-
-@pytest.fixture(scope="session")
-def s3_address(request):
-    return request.config.getoption("--s3-address")
-
-
-@pytest.fixture(scope="session")
-def no_tmp_build_dir(request):
-    return request.config.getoption("--no-tmp-build-dir")
-
-
-@pytest.fixture(scope="session")
-def test_variables(request):
-    return request.config.getoption("--test-variables")
-
+# Dirty hack for adding the module discoverable (for now)
+import os
+import sys
 
-@pytest.fixture(autouse=True)
-def commercial_test(request, bitbake_variables):
-    mark = request.node.get_closest_marker("commercial")
-    if mark is not None and not request.config.getoption("--commercial-tests"):
-        pytest.skip("Tests of commercial features are disabled.")
+cwd = os.getcwd()
+sys.path.append(cwd + "/image-tests/tests")
```

Comment:
Same thing as I [commented in mender-convert](https://github.com/mendersoftware/mender-convert/pull/262#discussion_r531133156). I think my suggestion is a bit better as it is independent on the cwd, but it is up to you. Most importantly is to be consistent :)

Reply:
Agreed! adopting